apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -330,6 +355,49 @@ export class SendMessagePush extends SendMessageBase {
     };
   }
 
+  private async updateOverridesWithInboxUnreadCount(
+    command: SendMessageChannelCommand,
+    overrides: Record<string, unknown>,
+    subscriber: SubscriberEntity,
+    inboxCountType: InboxCountTypeEnum
+  ): Promise<Record<string, unknown>> {
+    const filter =
+      inboxCountType === InboxCountTypeEnum.UNREAD ? { read: false, snoozed: false } : { seen: false, snoozed: false };
+
+    const inboxUnreadCount = await this.messageRepository.getCount(
+      command.environmentId,
+      subscriber._id,
+      ChannelTypeEnum.IN_APP,
+      filter,
+      { limit: 99 },
+      command.contextKeys
+    );
```

Comment:
@LetItRock what are your thoughts regarding using the `notifications-count` usecase? It comes with caching enabled, and thinking maybe just having a consitent one for it?